base/linalg/arnoldi.jl

```diff
@@ -382,14 +384,35 @@ iterations derived from [`eigs`](@ref).
 # Example
 
 ```jldoctest
-julia> A = spdiagm(1:4);
+julia> A = spdiagm(1:5);
+
+julia> Z = svds(A, nsv = 2)[1];
 
-julia> s = svds(A, nsv = 2)[1];
```

Comment:
Should `s` still appear in the first line of the docstring, despite its removal from this part?